components/TotalBudget.js

```diff
@@ -0,0 +1,102 @@
+import styled from "styled-components";
+
+export default function TotalBudget() {
+  return (
+    <>
+      <form>
+        <Fieldset>
+          <FlexContainer>
+            <label className="MediumFontStyle" htmlFor="totalBudget">
+              Gesamtbudget
+            </label>
```

Comment:
You could build a input coponent and so you could  outsource the label and input tag